docker-bake.hcl

```
@@ -154,7 +166,9 @@ function "javaversion" {
   params = [jdk]
   result = (equal(17, jdk)
     ? "${JAVA17_VERSION}"
-  : "${JAVA21_VERSION}")
+  : (equal(21, jdk)
+    ? "${JAVA21_VERSION}"
+  : "${JAVA25_VERSION}"))
```

we normally append preview to show its an early access Java image
```suggestion
  : "${JAVA25_VERSION}-preview"))
```

---

@gounthar  This `-preview` suffix can be added in the `tags` sections of the targets to avoid failure.

---

Reposted in a conversation not already hidden at https://github.com/jenkinsci/docker-agent/pull/939#discussion_r1955100055